webrtc-stats.html

```diff
@@ -1190,6 +1190,7 @@ <h2>
               </dt>
               <dd>
                 <p>
+                  Only [= map/exist =]s for video.
```

Comment:
Should this be consistent with "Does not [= map/exist =] for audio."

Reply:
I'll commit a change